ckanext/unaids/theme/templates/package/activity.html

```diff
@@ -0,0 +1,76 @@
+{% extends "package/read_base.html" %}
+
+{% block subtitle %}{{ _('Activity Stream') }} {{ g.template_title_delimiter }} {{ super() }}{% endblock %}
+
+{% block primary_content_inner %}
+<h1 class="hide-heading">
+  {% block page_heading %}
+  {{ _('Activity Stream') }}
+  {% endblock %}
+</h1>
+{% snippet 'snippets/activity_stream.html', activity_stream=activity_stream, id=id, object_type='package' %}
+
+<link rel="stylesheet" href="/bootstrap-table.css" />
+<div id="ActivityStreamLoading" class="text-center text-muted">
+  <h2>
+    <i class="fa fa-spinner fa-spin"></i>
+    <span> {{ _('Loading') }}</span>
+  </h2>
+</div>
+<div id="ActivityStreamContainer">
+  <table class="activity">
+    <thead>
+      <tr>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody></tbody>
+  </table>
+</div>
+
+<style>
+  ul.activity {
+    display: none;
+  }
+
+  #ActivityStreamContainer td {
+    border: none;
+    padding-bottom: 15px;
+  }
+</style>
+
+<script>
+  window.onload = function () {
+
+    // convert activity stream list into table
+    var table_rows = '';
+    $('ul.activity li').each(function (index) {
+      table_rows += '<tr><td class="item changed-package">';
+      table_rows += $(this).html();
+      table_rows += '</td></tr>';
+    });
+    $('#ActivityStreamContainer tbody').html(table_rows);
+
+    // convert table into paginated bootstrapTable
+    $('#ActivityStreamContainer table').bootstrapTable({
+      classes: 'table table-borderless',
+      showHeader: false,
+      pagination: true,
+      pageSize: 5,
```

Comment:
@tomeksabala - please confirm 5 is a good default

Reply:
It should be bigger. I think that current number of items displayed in the activity list is ok. You could use that.